bravado/mapping/param.py

```diff
@@ -108,3 +112,44 @@ def marshal_param(param, value, request):
         raise SwaggerMappingError(
             "Don't know how to marshal_param with location {0}".
             format(location))
+
+
+def add_file(param, value, request):
+    """Add a parameter of type 'file' to the given request.
+
+    :type param: :class;`bravado.mapping.param.Param`
+    :param value: The raw content of the file to be uploaded
+    :type request: dict
+    """
+    if request.get('file') is None:
+        # support multiple files by default by setting to an empty array
+        request['file'] = []
+
+        # The http client should take care of setting the content-type header
+        # to 'multipart/form-data'. Just verify that the swagger spec is
+        # conformant
+        expected_mime_type = 'multipart/form-data'
+        if expected_mime_type not in param.op.consumes:
+            raise SwaggerMappingError((
```

Comment:
I think you're right, this should be part of the static validation of the schema, performed by ssv.